en/html/README.md

```diff
@@ -208,6 +208,9 @@ $ git pull
 
 You'll need to substitute `<your-pythonanywhere-domain>` with your actual PythonAnywhere subdomain name, without the angle-brackets. Your subdomain name is normally your PythonAnywhere user name, but in some cases it might be a bit different (such as if your user name contains capital letters). So if this command doesn't work, use the `ls` (list files) command to find your actual subdomain/folder name, and then `cd` to there.
 
+Note that if you are still in the same Bash console used previously in this tutorial, you don’t have to use the `cd` command as you would already be in that directory.
```

Comment:
This seems unrelated to the other change or to the stated purpose of your pull request, so please file a separate pull request about this (and remove this change from this pull request), so it can be considered independently from the other change.

Reply:
Sure. I'll delete this file from the PR now.